lib/network/modules/InteractionHandler.js

```diff
@@ -511,7 +511,11 @@ class InteractionHandler {
       else if (event.detail) { /* Mozilla case. */
         // In Mozilla, sign of delta is different than in IE.
         // Also, delta is multiple of 3.
+        // For older FireFox versions
         delta = -event.detail / 3;
+      } //Modern FireFox
+      else if (event.deltaY) {
+        delta = -event.deltaY / 3;
```

Comment:
- Why negative?
- Why divided by 3?

In Firefox I get Y 3 with mode 1 (3 lines) when I scroll, in Chromium it's Y 53 with mode 0 (53 pixels). There should be some handling for this as the way it is now it may produce very different results in different browsers.

Also the modern standard API should be used as the default instead of using it as the last resort solution if everything else fails.